File: 최경열/algorithm1/300/나머지/Remainder.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});
let input = [];
rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  const n = input[0].split(" ").map(Number);
  const a = n[0];
  const b = n[1];
  const c = n[2];
  const one = (a + b) % c;
  const two = ((a % c) + (b % c)) % c;
  const three = (a * b) % c;
  const four = ((a % c) * (b % c)) % c;
  console.log(one);
  console.log(two);
  console.log(three);
  console.log(four);
});

File: 최경열/algorithm1/300/나머지/Remainder.py
a, b, c, = map(int, input().split(" "))

one = (a+b) % c
two = ((a % c) + (b % c)) % c
three = (a*b) % c
four = ((a % c) * (b % c)) % c

print(one)
print(two)
print(three)
print(four)
